provider/datagen/src/transform/cldr/decimal/mod.rs

```diff
@@ -72,7 +74,7 @@ impl crate::DatagenProvider {
             .collect())
     }
 
-    fn supported_locales(&self) -> Result<Vec<DataLocale>, DataError> {
+    fn supported_locales_for_numbers(&self) -> Result<HashSet<DataLocale>, DataError> {
```

Comment:
This poorly named function has been around for a long time and I hope it didn't cause confusion. It dates to the time when each transformer was implemented on its own struct.